test/argmax_benchmark.jl

```diff
@@ -0,0 +1,47 @@
+# Benchmark of argmax implementations
```

Comment:
Does this file need to be committed? It does not appear to be included in runtests.jl

Reply:
Removed it.